data/assets/examples/scale/timedependentscale/simple.asset

```diff
@@ -0,0 +1,29 @@
+-- This asset creates a SceneGraphNode that only displays coordinate axes, which grow at
+-- a speed of 1 m/s starting on January 1st, 2000 00:00:00. This means that on
+-- that date, the coordinate axes will disappear and, for example, on January 1st, 2000
+-- 12:00:00, the coordinate axes will be 43200 meters long.
+
+local Node = {
+  Identifier = "TimeDependentScale_Example_Simple",
+  Transform = {
+    Scale = {
+      Type = "TimeDependentScale",
+      ReferenceDate = "2000 JAN 01 00:00:00"
+    }
+  },
+  Renderable = {
+    Type = "RenderableCartesianAxes"
+  },
+  GUI = {
+    Name = "TimeDependentScale Example (Simple)",
```

Comment:
See previous PRs, about removing the word "simple". Still up for discussion though

Just "TimeDependentScale Example" is IMO good enough. And it is fine to have an advanced example as well